packages/components/nodes/chatmodels/ChatAnthropic/ChatAnthropic_LlamaIndex.ts

```diff
@@ -39,22 +39,22 @@ class ChatAnthropic_LlamaIndex_ChatModels implements INode {
                 options: [
                     {
                         label: 'claude-3-opus',
-                        name: 'claude-3-opus-20240229',
+                        name: 'claude-3-opus',
                         description: 'Most powerful model for highly complex tasks'
                     },
                     {
                         label: 'claude-3-sonnet',
-                        name: 'claude-3-sonnet-20240229',
+                        name: 'claude-3-sonnet',
```

Comment:
The model names are different in the LLamaIndex and Langchain Nodes.

Reply:
yes its a constants [here](https://github.com/run-llama/LlamaIndexTS/blob/main/packages/core/src/llm/LLM.ts#L631)